packages/stores/src/ui-config/trade-token-in-config.ts

```diff
@@ -410,6 +437,9 @@ export class ObservableTradeTokenInConfig extends AmountConfig {
     }
 
     if (this.amount) {
+      if (this._notEnoughLiquidity) return new NotEnoughLiquidityError();
```

Comment:
I think the no balance error should take precedence over the liquidity error. I see other DEXs do that, and IMO it's better optics and more accurate.